src/util/value.ts

```diff
@@ -82,6 +82,7 @@ export type EXPERIMENTAL_clipping = {
   heading?: number;
   pitch?: number;
   roll?: number;
+  keepAboveGround?: boolean;
```

Comment:
Our system's default is more or less don't allow entering ground. (In SceneProperty we have `allowEnterGround: boolean`).
For me it makes more sense to keep that perspective here too and calculate whether can enter the ground or not. What do you think? In your code it matters more about keeping above the ground, but that is the default state so not sure...

Reply:
I didn't know allowEnterGround property is exist. I fixed to use this property in [fix: use allowEnterGround](https://github.com/reearth/reearth-web/pull/357/commits/f70ee2e2ada51f9d1269bf37fbcc275671240a3a)